modules/uhsg_office_hours/src/OfficeHoursService.php

```diff
@@ -176,6 +210,28 @@ private function restructureOfficeHours(array $decodedBody) {
     return $officeHours;
   }
 
+  /**
+   * Sorts general office hours by language name using the current UI language.
+   *
+   * @param array $officeHours
+   */
+  private function sortGeneralOfficeHoursByLanguage(&$officeHours) {
+    $generalHours = $officeHours['general'];
+
+    usort($generalHours, function ($a, $b) {
+      $languageA = isset($a[0]['language']) ? $a[0]['language']->name->{$this->getCurrentLanguage()} : '';
+      $languageB = isset($b[0]['language']) ? $b[0]['language']->name->{$this->getCurrentLanguage()} : '';
```

Comment:
How this comparison works if we would receive `"language": { "name": { "fi": "Tämä objekti vain suomeksi" } }` while having english or swedish as an active language?

Reply:
According to our discussion, this causes notices which is OK behaviour. No fatal issues.